docs/annexes/changelog.md

```diff
@@ -2,6 +2,8 @@
 
 For release notes go to [Release Notes](release-notes.md). All the other (smaller) changes are documented here. This includes especially changes in wording and visuals. Improvements in the specification documented in changelog are not changing the functionality or compatibility of SAF. These changes only improve the understandability of SAF documentation.
 
+## 2.1.2023 Small description update
```

Comment:
For some reason. The changelog page does not build with the bullets. But it is not build with it even when build without this changes to changelog. I assume it will be build correctly after merge.
![image](https://user-images.githubusercontent.com/89633938/210230978-a8bbb30b-1e20-4687-8567-0b1aa6923913.png)